Route-mony/route-money-api
# #170259717 Add requester field

**What does this PR do?**

- Add requester filed in the model

**Description of Task to be completed?**

- Be able to save the identity of the requester

**How should this be manually tested?**

- Clone this repo using the command `git clone https://github.com/Route-mony/route-money-api.git`
- Create a local copy of this branch by running `git fetch ft-requester-field-170259717`
- Checkout to the branch `git checkout ft-requester-field-170259717`
- Run `python manage.py migrate`
- Check your request_model in your local database and see if it has requester column

**Any background context you want to provide?**

- We didn't have requester column

**What are the relevant pivotal tracker stories?**

- [#170259717](https://www.pivotaltracker.com/story/show/170259717)

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
